app/src/atoms/buttons/OnDeviceDisplay/__tests__/MediumButton.test.tsx

```diff
@@ -65,7 +65,7 @@ describe('MediumButton', () => {
   it('renders the tertiary light button', () => {
```

Comment:
Could you update the description?
```suggestion
  it('renders the tertiary low light button', () => {
```

Reply:
Done!